client/my-sites/checkout/checkout-thank-you/jetpack-plan-details.jsx

```diff
@@ -23,7 +22,7 @@ const BasicDetails = ( { translate } ) => (
 		description={ translate(
 			'We emailed you at %(email)s with information for setting up Akismet and VaultPress on your site. ' +
 				'Follow the instructions in the email to get started.',
-			{ args: { email: user.get().email } }
+			{ args: { email: user().get().email } }
```

Comment:
Should use Redux (and `useTranslate`)

Reply:
Done! (Well, the user part, at least)

Reply:
Just keep in mind that the Redux refactor can also be in a followup PR. This one is already big enough 🙂